data/002/0152/en_US.json

```
@@ -0,0 +1,13 @@
+{
+	"002": {
+		"0152": {
+			"name": "Custom Pretendo Error",
+			"message": "The online service is unavailable.\nPlease try again later.\n\nFor help, visit support.nintendo.com.",
+			"short_description": "Game server requires custom patches.",
+			"long_description": "The game server you are trying to connect to requires custom patches to protect its users against security vulnerabilities.",
+			"short_solution": "Download the latest version of Nimbus.",
+			"long_solution": "Since Nimbus 2.0.0, a new plugin containing custom patches for games is bundled with it. Download the latest version of Nimbus available to get the latest set of patches",
```

```suggestion
			"long_solution": "Since Nimbus 2.0.0, a new plugin containing custom patches for games is bundled with it. Download the latest version of Nimbus available to get the latest set of patches.",
```

This is my bad, I forgot to run the formatter